site/content/docs/5.2/components/modal.md

```diff
@@ -412,6 +419,13 @@ Add `.modal-dialog-centered` to `.modal-dialog` to vertically center the modal.
 
 Utilize the Boosted grid system within a modal by nesting `.container-fluid` within the `.modal-body`. Then, use the normal grid system classes as you would anywhere else.
 
+<!-- Boosted mod : design callout -->
+{{< ods-incompatibility-alert >}}
```

Comment:
Sure, the concept behind remains useful. Now, I would keep this callout because:
- even though it seems obvious and technically speaking it is possible to arrange the content as wanted, a modal content needs to follow the Orange design specifications, and this callout reminds that
- as we chose to add a callout before each variant that does not respect the design specifications, keeping this one would respect this "rule".

Reply:
We decided to delete this callout.